src/Raven.Server/Web/System/AdminTcpConnectionDebugInfoHandler.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.NetworkInformation;
+using System.Threading.Tasks;
+using Raven.Server.Routing;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Web.System
+{
+    public class AdminTcpConnectionDebugInfoHandler : RequestHandler
+    {
+        [RavenAction("/admin/debug/info/tcp/stats", "GET", AuthorizationStatus.Operator, IsDebugInformationEndpoint = true)]
+        public Task Statistics()
+        {
+            var properties = IPGlobalProperties.GetIPGlobalProperties();
+            var ipv4Stats = GetTcpStatisticsSafely(() => properties.GetTcpIPv4Statistics());
+            var ipv6Stats = GetTcpStatisticsSafely(() => properties.GetTcpIPv6Statistics());
+
+            var djv = new DynamicJsonValue
+            {
+                ["IPv4"] = ToDynamic(ipv4Stats),
+                ["IPv6"] = ToDynamic(ipv6Stats)
+            };
+
+            using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
+            using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                context.Write(writer, djv);
+            }
+
+            return Task.CompletedTask;
+
+            static DynamicJsonValue ToDynamic(TcpStatistics stats)
+            {
+                var result = new DynamicJsonValue();
+                if (stats == null)
+                    return result;
```

Comment:
shouldn't we return null here? Or a message that the platform is not supported?
On what platform isn't this supported?

Reply:
Done. Not sure which OS can throw this. Docs state it can throw that ex: https://docs.microsoft.com/en-us/dotnet/api/system.net.networkinformation.ipglobalproperties.gettcpipv6statistics?view=netcore-3.1

Win7 maybe?